src/AutoPilotPlugins/APM/APMSensorsComponentController.cc

```diff
@@ -274,9 +271,6 @@ void APMSensorsComponentController::_mavCommandResult(int vehicleId, int compone
                                      0,             // no delayed start
                                      0);            // no auto-reboot
 
-        } else {
-            // Onboard mag cal is not supported
-            _compassCal.startCalibration();
```

Comment:
@peterbarker This is what triggered the offboard cal. If the onboard was not accepted. So then this change doesn't have any dangling stuff left.